File: articles/algorithmic-risk-management-and-ai-governance/c/governance_readiness_score.c
#include <stdio.h>

int main(void) {
    double scores[] = {0.60, 0.62, 0.58, 0.52, 0.46, 0.50};
    double total = 0.0;
    for (int i = 0; i < 6; i++) total += scores[i];
    printf("governance_readiness_score=%.4f\n", total / 6.0);
    return 0;
}

File: articles/algorithmic-risk-management-and-ai-governance/rust/governance_readiness_score.rs
fn main() {
    let scores = [0.60_f64, 0.62_f64, 0.58_f64, 0.52_f64, 0.46_f64, 0.50_f64];
    let total: f64 = scores.iter().sum();
    let readiness = total / scores.len() as f64;
    println!("governance_readiness_score={:.4}", readiness);
}
